test/typecheck/main.go

```diff
@@ -79,6 +100,8 @@ func newAnalyzer(platform string) *analyzer {
 	platSplit := strings.Split(platform, "/")
 	ctx.GOOS, ctx.GOARCH = platSplit[0], platSplit[1]
 	ctx.CgoEnabled = true
+	tagsSplit := strings.Split(*tags, ",")
+	ctx.BuildTags = append(ctx.BuildTags, tagsSplit...)
```

Comment:
This will append an empty string if no tags are specified, right?

Reply:
yes, we should probably not do that, however I think it's harmless, go can't have a build constraint on a nothing tag.

Reply:
er no, because we split first, and it will be the empty slice

Reply:
`strings.Split("", ",")` => `[]`

Reply:
Are you sure?

https://play.golang.org/p/UcYudBHOokb

Reply:
bah, compare https://play.golang.org/p/Z4_2bLbJdWq which I tested :|